_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -38,11 +38,11 @@ The `object` element is not rendered for presentational purposes. If the `object
 
 ## Accessibility Support
 
-Non-supported media formats make screen readers render the text content of the element instead of other attributes.
+Non-supported media formats make screen readers render the text content of the element instead of other attributes. If there is no name for the `object` then the screen reader will not announce the context that is present visually.
 
 ## Background
 
-Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately.
+Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately. When the object resource is not loaded the fallback content is rendered. The only [accessible name][] that is announced by the screen reader is for an `object` that has valid `data` set. The `object` method of adding an [accessible name][] will depend on the role of the `data`.
```

Comment:
Shouldn't this be presented as a list?

Reply:
While Trevor's suggestion did improve the readability and so I fully support keeping it, it did not address my concern. 
I was literally talking about presentation. I believe this should be presented as a list instead of a paragraph. The first sentence is not related to the second. The second is not related to the third. The third and fourth are related (or, with Trevor's suggestion, they are merged into a single sentence). So, what I'm asking for is a list with three items, instead of a paragraph.

Reply:
Made into a list - thanks for the suggestion!